kv/util/codec/codec.go

```diff
@@ -0,0 +1,121 @@
+package codec
+
+import (
+	"encoding/binary"
+
+	"github.com/pingcap/errors"
+)
+
+const (
+	signMask uint64 = 0x8000000000000000
+
+	encGroupSize = 8
+	encMarker    = byte(0xFF)
+	encPad       = byte(0x0)
```

Comment:
I think using these consts makes the code harder to read, especially since the comments often refer to the fixed numbers.

Reply:
I think it's better to use a const number with accurate meaning instead of a magic number